
#include "centre_scene_handler.h"

///<<< BEGIN WRITING YOUR CODE
#include "muduo/base/Logging.h"
#include "scene/system/scene_system.h"
#include "proto/common/node.pb.h"
///<<< END WRITING YOUR CODE


void CentreSceneHandler::RegisterScene(::google::protobuf::RpcController* controller,const ::RegisterSceneRequest* request,
	::RegisterSceneResponse* response,
	::google::protobuf::Closure* done)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}




void CentreSceneHandler::UnRegisterScene(::google::protobuf::RpcController* controller,const ::UnRegisterSceneRequest* request,
	::Empty* response,
	::google::protobuf::Closure* done)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}



